src/sentry/integrations/github/client.py

```diff
@@ -118,7 +118,8 @@ def get_tree(self, repo_full_name: str, tree_sha: str) -> JSONData:
             elif msg == "Not Found":
                 logger.warning(f"The Github App does not have access to {repo_full_name}.")
             else:
-                logger.exception("An unknown error has ocurred.")
```

Comment:
We would simply log the issue and continue with all other repositories, thus, hammering Github.

Here's [an event](https://sentry.io/organizations/sentry/discover/sentry:e24949f714a549078b15b600b9e23f4e/?field=message&field=organization.slug&interval=1h&name=ApiError%3A+%7B%22message%22%3A%22Not+Found%22%2C%22documentation_url%22%3A%22https%3A%2F%2Fdocs.github.com%2Frest%2Freference%2Fapps%23create-an...&project=1&query=message%3A%22An+unknown+error+has+ocurred.+%7B%5C%22message%5C%22%3A%5C%22API+rate+limit+exceeded+for+installation+ID+865116.%5C%22%2C%5C%22documentation_url%5C%22%3A%5C%22https%3A%2F%2Fdocs.github.com%2Frest%2Foverview%2Fresources-in-the-rest-api%23rate-limiting%5C%22%7D+ApiError+sentry%2Fshared_integrations%2Fclient%2Fbase.py+_request+False+sentry.tasks.derive_code_mappings.derive_code_mappings%22+organization.slug%3Aadeo-0b&sort=-message&statsPeriod=3d&yAxis=count%28%29) for this.
<img width="231" alt="image" src="https://user-images.githubusercontent.com/44410/207718251-76eef03a-55d3-4b73-8923-521285a88e1c.png">